src/MessageBox.Avalonia/Extensions/SetStyle.cs

```diff
@@ -13,46 +13,46 @@ internal static void SetStyle(this Window window, Style style)
             switch (style)
             {
                 case Style.Windows:
-                {
-                    styles.Add(new StyleInclude(
-                            new Uri("avares://MessageBox.Avalonia/Styles/Windows/Windows.xaml"))
-                        {Source = new Uri("avares://MessageBox.Avalonia/Styles/Windows/Windows.xaml")});
-                    break;
-                }
+                    {
```

Comment:
More like an unnecessary change to the codestyle. These changes are best avoided as they are individual and increase cognitive load. Besides, it seems that it worse :) Please remove such changes